lerna.json

```
@@ -1,5 +1,5 @@
 {
-  "version": "7.0.0-next.25",
+  "version": "7.1.1-canary.1",
```

Can you back out this canary change?

```suggestion
  "version": "7.0.0-next.25",
```